scripts/artifacts-building/containers/artifact-build-chroot.yml

```diff
@@ -0,0 +1,160 @@
+---
+# Copyright 2016, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Create artifact container
+  hosts: localhost
+  connection: local
+  tasks:
+    #- name: Remove cache
```

Comment:
As far as I know this should be done to produce a clean base. Why is this remarked out?